docs/pages/versions/unversioned/sdk/application.md

```diff
@@ -0,0 +1,130 @@
+
+## Installation
+
+This API is pre-installed in [managed](../../introduction/managed-vs-bare/#managed-workflow) apps. To use it in a [bare](../../introduction/managed-vs-bare/#bare-workflow) React Native app, follow its [installation instructions](https://github.com/expo/expo/tree/master/packages/expo-device).
+
+## API
+
+```js
+import * as Application from 'expo-application';
+```
+
+### Constants
+
+- [`Application.applicationName`](#applicationapplicationname)
+- [`Application.bundleId`](#applicationbundleid)
+- [`Application.nativeAppVersion`](#applicationnativeappversion)
+- [`Application.nativeBuildVersion`](#applicationnativebuildversion)
+- [`Application.androidId`](#applicationandroidid) (Android only)
+
+### Methods
+
+- [`Application.getIosIdForVendorAsync()`](#applicationgetiosidforvendorasync) (iOS only)
+- [`Application.getInstallReferrerAsync()`](#applicationgetinstallreferrerasync) (Android only)
+- [`Application.getFirstInstallTimeAsync()`](#applicationgetfirstinstalltimeasync)
+- [`Application.getLastUpdateTimeAsync()`](#applicationgetlastupdatetimeasync) (Android only)
+
+## Constants
+
+### `Application.applicationName`
+
+The human-readable name of the application.
+
+- e.g., `Expo`, `Yelp`, `Instagram`
+
+### `Application.bundleId`
+
+The bundle ID of the application.
+
+- e.g., `com.cocoacasts.scribbles`, `com.apple.Pages`
+
+### `Application.nativeAppVersion`
+
+The `Info.plist` value for `CFBundleShortVersionString` on iOS and the version name set by `version` in app.json on Android at the time the native app was built.
+
+- e.g., `2.11.0`
+
+### `Application.nativeBuildVersion`
+
+The `Info.plist` value for `CFBundleVersion` on iOS (set with `ios.buildNumber` value in `app.json` in a standalone app) and the version code set by `android.versionCode` in app.json on Android at the time the native app was built.
+
+- e.g., `2.11.0`
+
+### `Application.androidId`
+
+**Android only.** A hexademical `string` unique to each combination of app-signing key, user, and device. The value may change if a factory reset is performed on the device or if an APK signing key changes. For more information about how the platform handles ANDROID_ID in Android 8.0 (API level 26) and higher, see [Android 8.0 Behavior Changes.](https://developer.android.com/about/versions/oreo/android-8.0-changes.html#privacy-all)
+
+Note: In versions of the platform lower than Android 8.0 (API level 26), this value remains constant for the lifetime of the user's device.
+
+See: https://developer.android.com/reference/android/provider/Settings.Secure.html#ANDROID_ID
+
+- e.g., `"dd96dec43fb81c97"`
+
+## Methods
+
+### `Application.getIosIdForVendorAsync()`
+
+**iOS only.** Gets the iOS identifier for vender [(IDFV)](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) value, an alphanumeric string that uniquely identifies a device to the app’s vendor.
```

Comment:
```suggestion
**iOS only.** Gets the iOS identifier for vendor [(IDFV)](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) value, an alphanumeric string that uniquely identifies a device to the app’s vendor.
```

Also need to document that, when, and why this could be null.